src/modules/layout/components/StyledToolbarItem.js

```diff
@@ -3,14 +3,18 @@
 import styled from 'styled-components'
 import CleanAnchor from '../../common/components/CleanAnchor'
 
-const StyledToolbarItem = styled(CleanAnchor)`
+export const StyledToolbarItem = styled(CleanAnchor)`
   display: inline-block;
   margin: 0 10px;
   padding: 8px;
   cursor: pointer;
-  color: ${props => props.theme.colors.textColor};
   border: none;
+  color: ${props => props.theme.colors.textColor};
   background-color: ${props => props.theme.colors.backgroundColor};
+  text-align: center;
+`
+export const SmallViewTip = styled.p`
```

Comment:
Move this to another file and keep the `export default`

Reply:
moved into a single file